src/coprocessor/dag/expr/builtin_string.rs

```diff
@@ -61,55 +62,43 @@ impl ScalarFunc {
         Ok(Some(input.len() as i64))
     }
 
+    #[inline]
+    fn find_str(text: &str, pattern: &str) -> Option<i64> {
```

Comment:
I prefer to provide `Option<usize>` for this function (from a non-business perspective) and then add `as i64` when calling it.

Reply:
Thx!